client/packages/invoices/src/DoseQtyColumn.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  ArrayUtils,
+  ColumnAlign,
+  ColumnDefinition,
+  RecordWithId,
+  UNDEFINED_STRING_VALUE,
+} from '@openmsupply-client/common';
+import { ItemRowFragment } from '@openmsupply-client/system';
+import { InboundLineFragment } from './InboundShipment';
+import { StockOutLineFragment } from './StockOut';
+
+export const getDosesQuantityColumn = <
```

Comment:
Why this one in invoice and not common?

Reply:
Not being used outside of invoices like the doses per unit right now... can move to common if it does get used in other areas 👀